substrait/proto/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+# SPDX-License-Identifier: Apache-2.0
+
+find_package(Protobuf REQUIRED)
+include_directories(${Protobuf_INCLUDE_DIRS})
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
+
+
+SET(PROTO_INCLUDE_DIR "${CMAKE_SOURCE_DIR}/third_party/substrait/proto/substrait")
+cmake_path(GET PROTO_INCLUDE_DIR PARENT_PATH PROTO_INCLUDE_TOPLEVEL_DIR)
+SET(PROTOBUF_IMPORT_DIRS ${PROTO_INCLUDE_DIR}/extensions)
```

Comment:
Do you have any extensions?

Reply:
I do not.   One of the substrait proto definition files in an extensions subdirectory:

https://github.com/substrait-io/substrait/blob/main/proto/substrait/extensions/extensions.proto